Dev ops team

1jc opened the pull request:
1jc Joette Damo Developer creates folder within DevOps_Team branch which allows the DevOps team members to upload for their codes for issues/tasks in their own respective subfolder; likewise should not interfere with each DevOps_Team member PRs.

To create a branch in development branch for devops:
1.	At the top of the app, click current branch and then in the list of branches, click the branch that you want to base your new branch on.
2.	Click new branch.
3.	Under name, type the name of the new branch.
4.	Use the drop-down to choose a base branch for your new branch.
5.	Click create branch.
To create a folder for the DevOps source code in the repository.
1.	Click on create new file.
2.	Enter the folder name that you want, then press /
3.	Enter a sample file name. You must enter some text.
4.	Click commit new file to create the folder.
5.	Your folder is created.
Team members in DevOps_Team:
1.	Nathaniel Ceres
2.	Joette Damo
3.	Dagmawi Demissie
4.	Michael Glanz
5.	Jonathan Kindle
6.	Bikash Thapa
To create a subfolder in DevOps folder for each of the team members. 
1.	Go to the folder inside which you want to create another folder.
2.	Click on new file.
3.	On the text field for the file name, first write the folder name you want to create.
4.	Then type /. …
5.	You can add more folders similarly.
6.	Finally give the new file a name.

Sample files to be added:
. gitignore
LICENSE
README.md
Flopp-data-export.py

For each Team Member added Documentation and Activities Documentation file

1jc committed:
Create Requirements for folder

1jc Joette Damo Developer creating requirements for DevOps_Team folder.  Made new branch for team as DevOps_Team.  Also, made folder for DevOps_Team as DevOps.  This file called "Requirements for folder".

1jc committed:
Create .gitignore

1jc Joette Damo Developer creating .gitignore file.

1jc committed:
Create LICENSE

1jc Joette Damo Developer creating a LICENSE file

1jc committed:
Create README.md

Team members in DevOps_Team:
1.	Nathaniel Ceres
2.	Joette Damo
3.	Dagmawi Demissie
4.	Michael Glanz
5.	Jonathan Kindle
6.	Bikash Thapa

What you need to do for issue/tasks for a grade in AD440

1jc committed:
Create Documentation

1jc Joette Damo Developer created for Nathaniel_Ceres Documentation file.

1jc committed:
Create Activities Documentation

1jc Joette Damo Developer created for Nathaniel_Ceres Activities Documentation file

1jc committed:
Create Documentation

1jc Joette Damo Developer created Documentation file

1jc committed:
Create Activities Documentation

1jc Joette Damo created Activities Documentation file

1jc committed:
Create Documentation

1jc Joette Damo Developer created Created Dagmawi_Demissie Documentation file

1jc committed:
Create Activities Documentation

1jc Joette Damo Developer created Create Dagmawi_Demissie Activities Documentation file

1jc committed:
Create Documentation

1jc Joette Damo Developer created Michael_Glanz Documentation file

1jc committed:
Create Activities Documentation

1jc Joette Damo created Michael_Glanz Activities Documentation file

1jc committed:
Create Documentation

1jc Joette Damo Developer created for Jonathan_Kindle Documentation file

1jc committed:
Create Activities Documentation

1jc Joette Damo Developer created Jonathan_Kindle Activities Documentation file

1jc committed:
Create Documentation

1jc Joette Damo Developer created for Create for Bikash_Thapa Documentation file

1jc committed:
Create Activities Documentation

1jc Joette Damo created for Bikash_Thapa Activities Documentation file

1jc commented:
### Activities Documentation:

Date Activity Time Spent
Jan 13 Assign myself to issue move to "To Do" 1 hour
Jan 13 Research 1 hour
Jan 25 Research 1 hour
Jan 27 Research 1 hour
Jan 27 prototyping, implementation, deployment
implementation, task management 5 hours

1jc commented:
### **Implementation and Documentation Criteria:**
1jc Joette Damo Developer creates folder within DevOps_Team branch which allows the DevOps team members to upload for their code issues/tasks in their own respective subfolder; likewise should not interfere with each DevOps_Team member PR's.

To create a branch in development branch for devops:

At the top of the app, click current branch and then in the list of branches, click the branch that you want to base your new branch on.
Click new branch.
Under name, type the name of the new branch.
Use the drop-down to choose a base branch for your new branch.
Click create branch.
To create a folder for the DevOps source code in the repository.
Click on create new file.
Enter the folder name that you want, then press /
Enter a sample file name. You must enter some text.
Click commit new file to create the folder.
Your folder is created.
Team members in DevOps_Team:
Nathaniel Ceres
Joette Damo
Dagmawi Demissie
Michael Glanz
Jonathan Kindle
Bikash Thapa
To create a subfolder in DevOps folder for each of the team members.
Go to the folder inside which you want to create another folder.
Click on new file.
On the text field for the file name, first write the folder name you want to create.
Then type /. …
You can add more folders similarly.
Finally give the new file a name.
Sample files to be added:
. gitignore
LICENSE
README.md
Flopp-data-export.py
Creating for each Team Member in their subfolders Documentation and Activities Documentation files.

eknigge commented on DevOps/.gitignore:
```diff
@@ -0,0 +1,129 @@
+dmypy.json
+
+# Pyre type checker
+.pyre/
```
There are a lot of ignored files, far more than currently exist in the repository. I think this file can be simplified for now, and updated as the product develops.

1jc replied:
This applies to the DevOps_Team Eric Knigge is part of the Front-end team

eknigge commented on DevOps/LICENSE:
```diff
@@ -0,0 +1,201 @@
+   Apache License
```
Including a license for our project is probably a good idea, but it should be for the whole project. Recommend removing it here.

1jc replied:
This applies to the DevOps_Team Eric Kniggie is part of the Front-End team

eknigge commented on DevOps/README.md:
```diff
@@ -0,0 +1,49 @@
+Team members in DevOps_Team:
```
Including team members, their responsibilities, and other documentation is a good idea, but most of this file is a repeat of the grading criteria. Recommend removing that part.

1jc replied:
This applies to the DevOps Team Eric Kniggie is part of the Front-end team

eknigge reviewed (changes requested):
There are several issues with this request. First, it's unclear what the scope and intent is, since there is no overview, descriptive title, or other details provided. Second, there are a multitude of new files and directories introduced, which is concerning. The project is meant to be collaborative, and team members within a group will work on different aspects, so creating documentation in isolation, won't be effective or productive. 

These issues need to be addressed before this can be approved.

dagmawidemissie17 reviewed (approved):
Remove the **license** and **gitignore** files for now to avoid future merge conflicts with the main branch. Also, one documentation file is enough for each of our folders? The rest looks good. Thx!

1jc committed:
Delete .gitignore

1jc Joette Damo deleted .gitignore file

1jc committed:
Delete LICENSE

1jc Joette Damo Developer delete LICENSE file

1jc committed:
Delete Activities Documentation

1jc Joette Damo Developer deleted Activities Documentation file

1jc committed:
Delete Activities Documentation

1jc Joette Damo deleted Activities Documentation file

1jc committed:
Delete Activities Documentation

1jc Joette Damo Developer deleted Activities Documentation file

1jc committed:
Delete Activities Documentation

1jc Joette Damo Developer deleted Jonathan_Kindle Activities Documentation file

1jc committed:
Delete Activities Documentation

1jc Joette Damo Developer deleted Michael_Glanz Activities Documentation file

1jc committed:
Delete Activities Documentation

1jc Joette Damo Developer deleted Nathaniel_Ceres Activities Documentation file

1jc reviewed (commented):
1jc Joette Damo remove the following files:
LICENSE
.gitignore
removed Activities Documentation files for Bikash_Thapa, Dagnawi_Demissie, Joette Damo, Jonathan_Kindle, Michael_Glanz, Nathaniel_Ceres

eknigge reviewed (approved):
Looks much better, thanks for making the changes. There appear to be a lot of commits to your file, you may want to look into squash those into a single commit before uploading to GitHub in the future.

1jc commented:
1jc Joette Damo Developer will not delete DevOps_Team branch.  Instructor asked why subfolders were in Team members' names.  I told instructor that I had asked him along members of both classes Tuesday and Thursday about criteria of putting team members' names as names on the subfolders.  I ran this post in Slack and received no advise nor reaction comments from instructor nor members in the General Slack channel for the AD440 class.  This is being addressed by another team member of the DevOps_Team.  Michael Glanz team member was instructed by instructor to address this privacy issue to privacy@github.com since this was an issue with him.  Again, I had asked my team members about criteria of team members' on subfolders for github - no one gave me advisement nor reacted to helping me on this.

1jc commented:
1jc Joette Damo gave additional information to team members about uploading files to DevOps_Team folder to subfolder in following message:
Hello Team Members - there is an issue about subfolders made in team members' names.  When uploading your code to the DevOps_Team branch upload to folder inside the branch named DevOps then make a subfolder with the issue name & number.   Thanks @Joette Damo
